roles/zuul/templates/etc/zuul/zuul.conf

```diff
@@ -19,24 +19,38 @@ start = {{ zuul_gearman_server_start }}
 
 {% endfor %}
 
+{% if not zuul_zuul_v3 %}
 {% for name, details in zuul_sites.items() %}
 [site "{{ name }}"]
 {% for key, value in details.items() %}
 {{ key }} = {{ value }}
 {% endfor %}
 
 {% endfor %}
+{% endif %}
 
 [zuul]
+state_dir = /var/lib/zuul
+url_pattern = {{ zuul_logs_url }}/{build.parameters[LOG_PATH]}
+{% if zuul_zuul_v3 %}
+log_config = /etc/zuul/scheduler-logging.conf
+pidfile = /var/run/zuul-scheduler/zuul-sscheduler.pid
+tenant_config = /etc/zuul/tenant.yaml
+zookeeper_hosts = {{ zuul_zookeeper_hosts }}
+
+[executor]
+log_config = /etc/zuul/launcher-logging.conf
+
+[launcher]
```

Comment:
yea, i'm assuming it's still a transitional thing but there are still a couple of variables that get pulled from launcher